lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1670,7 +1670,7 @@ def test_certificate_generation_for_students(self):
             'failed': 3,
             'skipped': 2
         }
-        with self.assertNumQueries(175):
+        with self.assertNumQueries(191):
```

Comment:
Hmm.  Will we need to do work on making cert generation more performant in a follow-up?  Or are we cool with letting that be slow for now?